packages/react-components/src/components/ReactionButton.tsx

```diff
@@ -176,7 +177,7 @@ export const ReactionButton = (props: ReactionButtonProps): JSX.Element => {
         <ControlBarButton
           {...props}
           componentRef={reactionButtonRef}
-          className={mergeStyles(styles, props.styles)}
+          styles={concatStyleSets(styles, props.styles)}
```

Comment:
This change is to prevent a 'Maximum call stack size exceeded' error when returning from a breakout room to main meeting